test/dev-app/views/test/elastic.lp

```diff
@@ -0,0 +1,48 @@
+<?lua local form = require "sailor.form" ?>
+<form method="post">
+            <h4>Search box</h4>
+            <input type="text" id="search" name="search" />
+            <input type="submit" class="btn btn-primary btn-md"/>
+            </form>
+            <?lua 
+            function table.val_to_str ( v )
```

Comment:
The elasticsearch-lua client gives out a nested table. I convert this table into a string so that I can print it on the screen using page:write(). 

In the beginning of the file, I also create an input box for entering search terms. The result is printed on the screen. 

This file is meant to be an example for testing ES models.